protos-frontend/frontend/FrontendService.proto

```diff
@@ -59,4 +64,32 @@ message TriggerEmailVerificationRequest {
 }
 message TriggerEmailVerificationResponse {
     enums.VerificationResponseType response_type = 1;
+}
+
+//Pan verification
+message GetPanVerificationScreenRequest{
+
+}
+
+
+message GetPanVerificationScreenResponse {
+    enums.VerificationResponseType response_type = 1;
+    string image_url = 2;
+    string pan_screen_title = 3;
```

Comment:
this is coming inside `GetPanVerificationScreenResponse` so title alone can suffice.